packages/web/components/Player/PlayerGuild.tsx

```diff
@@ -35,11 +36,21 @@ export const InternalGuildLink: React.FC<InternalGuildLinkProps> = ({
 
 type DaoHausLinkProps = {
   daoUrl: string | null;
+  linkProps?: Maybe<LinkProps>;
 };
```

Comment:
imo it would be cleaner to send linkProps as part of the props itself. 
```
type DaoHausLinkProps = { daoUrl: string | null } & LinkProps;
```
and we can destrucute the remaining props as already done below.

Reply:
Taken care of during the rebase